source/features/link-issues-pr-user-profile.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import elementReady from 'element-ready';
+import issueIcon from 'octicon/issue-opened.svg';
+import pullRequestIcon from 'octicon/git-pull-request.svg';
+
+async function init(): Promise<false | void> {
+	const repositoryTab = await elementReady('[data-filterable-for="your-repos-filter"]');
+	if (!repositoryTab) {
```

Comment:
This specific condition could be part of `page-detect` because it can be detected by URL

Reply:
So make a new one for it?

By the top repo we did this